data/src/main/java/com/aconno/sensorics/data/repository/format/FormatRepositoryImpl.kt

```diff
@@ -0,0 +1,42 @@
+package com.aconno.sensorics.data.repository.format
+
+import com.aconno.sensorics.domain.format.AdvertisementFormat
+import com.aconno.sensorics.domain.format.AdvertisementFormatJsonConverter
+import com.aconno.sensorics.domain.repository.FormatRepository
+
+class FormatRepositoryImpl(
```

Comment:
Maybe OnDeviceRepository so it doesn't class with the other advertisement repository?

Reply:
Renamed to LocalFormatRepositoryImpl.